config/convert_test.go

```diff
@@ -216,7 +216,7 @@ func TestFileModePresent(t *testing.T) {
 		},
 		{
 			"present",
-			FileMode(0644),
+			FileMode(0o644),
```

Comment:
I wasn't familiar with this file mode. I looked it up and it seems like it is used for Python 3. Since this repo is in Go, I'm not sure if I'm missing something. Could you share some context around this change?

Reply:
It's not just Python 3.  Since [Go 1.13](https://go.dev/doc/go1.13#language) both formats for octal literals [are allowed](https://go.dev/ref/spec#Integer_literals) and are equivalent.

Reply:
This was not a direct change but instead a result of running `gofumpt`.